### PR TITLE
INTEL-3589 ticket resolution

### PR DESCRIPTION
**Pull request by GSQ-AI**
Addressing https://itp-qld.atlassian.net/browse/INTEL-3589

Remove "unitless" from operational reporting collections (Approved by Mike to remove from collections rather than the concept as a whole, to prevent orphaned records using "unitless" as well as to allow its use in other systems).
Replace "bauxite" with "alumina" in the bulk shipping ore collection (Approved by Mike to use "alumina" instead of “Alumina (Al2O3)“ so as to use the existing alumina concept).
Add "Chryoscolla" concept.
Add "Chrysocolla" concept to the gemstone collections.
Add "Nickel Ore" concept (Approved by Mike to use "Nickel Ore" rather than "Nickel Lateritic Ore", as the system demands that the product form declares itself "lateritic", and the use of "Nickel Ore" instead brings itself in alignment with the other direct shipping ores (Iron Ore, etc.) as well as makes it reusable in other circumstances.)
Add "Nickel Ore" concept to the bulk shipping ore collection.

### CODE REVIEW

File: vocabularies-gsq/geo-commodities.ttl [hunk 1]
```diff
@@ -804,12 +806,13 @@ cdt:material-directshippingore
...
     skos:definition "The commodities relevant, applicable, and selectable as part of the material type direct shipping ore."@en ;
     skos:member
-        cdt:bauxite ,
+        cdt:alumina ,
         cdt:cassiterite-ore ,
         cdt:hematite-ore ,
         cdt:iron-ore ,
         cdt:magnetite-ore ,
-        cdt:manganese-ore ;
+        cdt:manganese-ore
```
Comment (Steven-Bowden) [inline vocabularies-gsq/geo-commodities.ttl:814]: comma?

**Review by Steven-Bowden — CHANGES_REQUESTED** [review]
do we need a comma on ln 812?

File: vocabularies-gsq/geo-commodities.ttl (hunk 1 above)
Comment (GSQ-AI) [reply vocabularies-gsq/geo-commodities.ttl]: Yes. Repaired now.

**Comment by GSQ-AI** [comment]
> do we need a comma on ln 812?

Yes we did, I've repaired this and updated the commit.

**Review by Steven-Bowden — APPROVED** [review]
looks good

**Review by LukeHauck — APPROVED** [review]
Looks good, approved.

**Pull request merged**